core/templates/dev/head/pages/story-viewer-page/chapters-list/story-viewer-chapters-list.directive.ts

```diff
@@ -85,26 +87,53 @@ angular.module('oppia').directive('storyViewerChaptersList', [
 
           $anchorScroll.yOffset = -80;
 
-          ctrl.setIconHighlight = function(index) {
-            ctrl.activeHighlightedIconIndex = index;
+          ctrl.isVisibleCard = function(storyNode) {
+            if (!storyNode) {
+              return false;
+            }
+            var isNextPendingNode = (
+              ctrl.storyPlaythroughObject.getNextPendingNodeId() ===
+              storyNode.getId());
+            return storyNode.isCompleted() || isNextPendingNode;
           };
 
-          ctrl.unsetIconHighlight = function() {
-            ctrl.activeHighlightedIconIndex = -1;
+          ctrl.getIconTitle = function(node, isMobile, index) {
+            var prefix = '';
+            if (isMobile) {
```

Comment:
This could use some explanation (in a comment).

Reply:
Removed this function.